packages/client/composer.json

```diff
@@ -0,0 +1,32 @@
+{
+	"name": "automattic/jetpack-client",
+	"description": "Jetpack Client",
```

Comment:
Can we be a bit more descriptive here?

Maybe `A client for performing authorized remote requests.` or something.